rpc/params/apierror.go

```diff
@@ -214,6 +214,45 @@ const (
 	CodeNotValid                  = "not valid"
 )
 
+// TranslateWellKnownError translates well known wire error codes into a github.com/juju/errors error
+// that matches the error code.
+func TranslateWellKnownError(err error) error {
+	code := ErrCode(err)
+	switch code {
+	// case CodeDeadlineExceeded:
+	// 	return errors.NewTimeout(err, "")
```

Comment:
?

Reply:
errors package doesn't have a deadline exceeded, not sure timeout is the best error, will add it as a todo